src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -8702,8 +8703,9 @@ namespace HVACVariableRefrigerantFlow {
         if ((Garate > 0.0) && ((!VRF(VRFNum).HeatRecoveryUsed && CoolingLoad(VRFNum)) ||
                                (VRF(VRFNum).HeatRecoveryUsed && TerminalUnitList(TUListIndex).HRCoolRequest(IndexToTUInTUList)))) {
             // 1.1) Cooling coil is running
-            QZnReqSenCoolingLoad = max(0.0, -1.0 * ZoneSysEnergyDemand(ZoneIndex).OutputRequiredToCoolingSP);
-            Tout = T_TU_in - QZnReqSenCoolingLoad * 1.2 / Garate / 1005;
+            //QZnReqSenCoolingLoad = max(0.0, -1.0 * ZoneSysEnergyDemand(ZoneIndex).OutputRequiredToCoolingSP);
```

Comment:
Here's the big question. What Tout is needed here? The Tout when the coil is operating I assume. Likne 8706 is wrong since for that zone, the TU operated and probably met the load, so OutputRequiredToCoolingSP is probably 0. Instead I use the actual coil outlet temp, but this is premised on needing actual Tout.

Reply:
I guess I should read the Eng. Ref. to get better acquainted.
@nealkruis @mbadams5 @mjwitte @Myoldmopar @jasondegraw @kbenne this will need some serious thought and review because I don't know this model. What the heck should it do? Does it need full output conditions while coils are operating? Yet the coils can have different PLR's so how to average correctly to get condenser performance? Anyone recognize old line 8706 above? Q*1.2/Mdot/1005 ? Figuring out what this function needs will probably correct this issue. If Tout at full air flow is really needed then the coils will need to be simulated again, or better yet that information needs to be saved when the TUs model full flow and then used here. Is Xiufeng Pang or RP Zhang or anyone at LBNL available for consult on this model?